homeworks/25/chat-system/src/main/java/org/elsys_bg/sockets_hw/Server.java

```diff
@@ -0,0 +1,111 @@
+package org.elsys_bg.sockets_hw;
+
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+public class Server {
+
+	private static List<Socket> clientSockets = new ArrayList<>();
+	private static Map<Integer, String> clientNames = new HashMap<>();
+
+	public static void main(String argv[]) {
+
+        DateTimeFormatter dtf = DateTimeFormatter.ofPattern("hh:mm a");  
+        LocalDateTime now = LocalDateTime.now();
```

Comment:
The time will be set at the moment of starting the server and it won't change. The message should be printed with the time of their sending. The time should be taken/updated for every message.